packages/components/src/custom-gradient-picker/gradient-bar/index.tsx

```diff
@@ -74,19 +86,27 @@ export default function CustomGradientBar( {
 	disableInserter = false,
 	disableAlpha = false,
 	__experimentalIsRenderedInSidebar,
-} ) {
-	const gradientMarkersContainerDomRef = useRef();
+}: CustomGradientBarProps ) {
+	const gradientMarkersContainerDomRef: LegacyRef< HTMLDivElement > =
+		useRef( null );
 
 	const [ gradientBarState, gradientBarStateDispatch ] = useReducer(
 		customGradientBarReducer,
 		customGradientBarReducerInitialState
 	);
-	const onMouseEnterAndMove = ( event ) => {
+	const onMouseEnterAndMove: MouseEventHandler< HTMLDivElement > = (
+		event
+	) => {
 		const insertPosition = getHorizontalRelativeGradientPosition(
 			event.clientX,
 			gradientMarkersContainerDomRef.current
 		);
 
+		// NTS: `getHorizontalRelativeGradientPosition` will return `undefined` if the `containerElement` arg is falsey.
+		if ( insertPosition === undefined ) {
+			return;
+		}
```

Comment:
`getHorizontalRelativeGradientPosition` will return `undefined` if the `containerElement` arg is falsey